convert/tests/text/test_text_wrapper.py

```diff
@@ -78,7 +78,7 @@ def test_wrap_paragraph_with_long_word_at_the_beginning():
     assert result == ["very_long_", "word word1 ", "word2"]
 
 
-def test_wrap_paragraph_with_word_multiple_times_longer_then_string():
+def test_wrap_paragraph_with_word_multiple_times_longer_then_string() -> None:
     tw = TextWrapper(line_length=10)
     text = "word1 very_long_word_which_can't_be_fitted_even_in_two_or_three_lines word2 word3"  # noqa: E501
```

Comment:
could you remove this and any flake8 comments in convert service?
```suggestion
    text = "word1 very_long_word_which_can't_be_fitted_even_in_two_or_three_lines word2 word3"
```

Reply:
Done